convene-web/app/models/feature.rb

```diff
@@ -8,6 +8,7 @@ def self.enabled?(feature)
     return true if feature == :demo && ENV['DEMO_ENABLED']
     return true if feature == :system_test && ENV['SYSTEM_TEST']
     return true if feature == :configure_room && ENV['CONFIGURE_ROOM']
+    return true if feature == :identification && ENV['IDENTIFICATION']
```

Comment:
I'm beginning to get a bit worried that we're putting a so much configuration into distinct ENV variables; perhaps it's worth turning the Feature model into a "real model" that stores features in a comma-separated list or something? Not urgent by any means; but may be worth adding a `TODO` to the codebase for someone who is interested in doing some of that kind of work!